gc/base/HeapMap.cpp

```diff
@@ -214,7 +214,7 @@ MM_HeapMap::numberBitsInRange(MM_EnvironmentBase *env, void *lowAddress, void *h
 	uintptr_t baseIndex, topIndex;
 	
 	/* Validate passed heap references */
-	Assert_MM_true(lowAddress < highAddress);
+	Assert_MM_true(lowAddress <= highAddress);
```

Comment:
I restored the assert to the original, and added a check in the caller (MM_ConcurrentGC::determineInitWork) to skip empty region.